ci/pipeline-haproxy-maintenance.yml

```diff
@@ -430,7 +430,7 @@ resources:
     source :
       driver:          gcs
       bucket:          haproxy-boshrelease
-      key:             version-11.16
+      key:             Version-11.16
```

Comment:
shouldn't the name be `maintenance-version` or similar to avoid manual changes to the key once we move the maintenance version up?

Reply:
discussed with @maxmoehl , @peanball , @Mrizwanshaik. As the key we will use schema: version-<major>, e.g. version-11, version-12.

Reply:
@Mrizwanshaik Could you please adapt the key to `version-11`